Shop-1/src/index.ts

```diff
@@ -0,0 +1,21 @@
+import express from "express";
+import cors from "cors";
+const app = express();
+app.use(cors());
+app.use(express.json());
+
+app.use("/products", products);
```

Comment:
I'm getting errors on products and users. You need to import them from the route files. 

import products from "../src/routes/productsRouter";
import users from "../src/routes/usersRouter";